site/en/docs/extensions/reference/commands/index.md

```diff
@@ -8,89 +8,159 @@ You must have a `"manifest_version"` of at least `2` to use this API.
-the extension to be uninstallable.
+: An optional property that declares default keyboard shortcuts for the command. If omitted, the
+command will be unbound. This property can either take a string or an object value.
 
-\* Additionally, on Chrome OS, you can specify 'Search' as a modifier.
+: A string values specifies the default keyboard shortcut that should be used across all platforms.
 
-Certain Chrome shortcuts (e.g. window management) always take priority over Extension Command
-shortcuts and can not be overwritten.
+: An object value allows the extension developer to customize the keyboard shortcut for each
+platform. When providing platform-specific shortcuts, valid object properties are `default`,
+`chromeos`, `linux`, `mac`, and `windows`.
+
+: See [Key combination requirements][header-key-combos] for
+additional details.
+
+`description`
+
+: A string used to provide the user with a short description of the command's purpose. This string
+appears in extension keyboard shortcut management UI. Descriptions are required for standard
+commands, but are ignored for [Action commands][header-action].
+
+An extension can have many commands, but may specify at most 4 suggested keyboard shortcuts. The
+user can manually add more shortcuts from the `chrome://extensions/shortcuts` dialog.
+
+### Supported Keys
+
+The following keys are usable command shortcuts. Key definitions are case sensitive. Attempting to
+load an extension with an incorrectly cased key will result in a manifest parse error at
+installation time.
+
+Alpha keys
+
+: `A` … `Z`
+
+Numeric keys
+
+: `0` … `9`
+
+Standard key strings
+
+: General–`Comma`, `Period`, `Home`, `End`, `PageUp`, `PageDown`, `Space`, `Insert`, `Delete`
+
+: Arrow keys–`Up`, `Down`, `Left`, `Right`
+
+: Media Keys–`MediaNextTrack`, `MediaPlayPause`, `MediaPrevTrack`, `MediaStop`
+
+Modifier key strings
+
+: `Ctrl`, `Alt`, `Shift`, `MacCtrl` (macOS only), `Command` (macOS only), `Search` (Chrome OS only)
+
+{% Aside 'note' %}
+
+`Tab` was removed from list of supported keys in Chrome 33 for accessibility reasons.
+
+{% endAside %}
+
+### Key combination requirements {: #key-combinations }
+
+- Extension Command shortcuts must include either `Ctrl`\* or `Alt`.
+
+    - Modifiers **cannot** be used in combination with Media Keys.
+
+- \* On macOS `Ctrl` is automatically converted into `Command`.
+
+    - To use the Control key on macOS, replace `Ctrl` with `MacCtrl` when defining the `"mac"`
+      shortcut.
+
+    - Including `MacCtrl` in other shortcuts will cause the extension to be uninstallable.
+
+- `Shift` is an optional modifier on all platforms.
+
+- `Search` is an optional modifier exclusive to Chrome OS.
+
+- Certain operating system and Chrome shortcuts (e.g. window management) always take priority over
+  Extension Command shortcuts and can not be overwritten.
+
+{% Aside 'note' %}
+
+Key combinations that involve `Ctrl+Alt` are not permitted in order to avoid conflicts with the
+`AltGr` key.
```

Comment:
Is `AltGr` a real key name?

Reply:
[It is](https://en.wikipedia.org/wiki/AltGr_key). I was also unfamiliar with it before I encountered it in this document.